.changeset/honest-seas-rush.md

```diff
@@ -0,0 +1,6 @@
+---
+"@uploadthing/react": patch
+"@uploadthing/solid": patch
+---
+
+feat: generateSolidComponents & generateReactComponents functions
```

Comment:
```suggestion
feat: `generateComponents` functions for solid and react allows to pass the generic `FileRouter` once instead of for everytime the component is used
```